keyboards/protokeeb/rev1/keyboard.json

```diff
@@ -0,0 +1,507 @@
+                {
+                    "label": "/",
+                    "matrix": [0, 3],
+                    "x": 3,
+                    "y": 0,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "4",
+                    "matrix": [1, 0],
+                    "x": 0,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "5",
+                    "matrix": [1, 1],
+                    "x": 1,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "6",
+                    "matrix": [1, 2],
+                    "x": 2,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "*",
+                    "matrix": [1, 3],
+                    "x": 3,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "1",
+                    "matrix": [2, 0],
+                    "x": 0,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "2",
+                    "matrix": [2, 1],
+                    "x": 1,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "3",
+                    "matrix": [2, 2],
+                    "x": 2,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "-",
+                    "matrix": [2, 3],
+                    "x": 3,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "0",
+                    "matrix": [3, 0],
+                    "x": 0,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": ".",
+                    "matrix": [3, 1],
+                    "x": 1,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "ENTER",
+                    "matrix": [3, 2],
+                    "x": 2,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "+",
+                    "matrix": [3, 3],
+                    "x": 3,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                }
+            ]
+        },
```

Comment:
Given that these layouts are all functionally identical, only one should be included.
```suggestion
```

Reply:
Ok. I have removed the "LAYOUT" layout from the "keyboard.json" file as requested. Also, I have updated the default keymap's "keymap.c" file as the "LAYOUT" was being used in that file.